client/payment-details/summary/index.tsx

```diff
@@ -177,6 +186,11 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 
 	const isFraudOutcomeReview = isOnHoldByFraudTools( charge, paymentIntent );
 
+	const disputeFee = charge.dispute && getDisputeFee( charge.dispute );
+	const transactionFee = charge.balance_transaction
+		? charge.balance_transaction.fee
+		: charge.application_fee_amount;
```

Comment:
I don't understand this part. Could you explain?

Reply:
Good catch!

I understood the need for this approach by observing the following ternary statement introduced in the commit 7b4b9ba.

https://github.com/Automattic/woocommerce-payments/blob/648f0a1e1116e2f621ac7a0a69bb43bbded6b242/client/utils/charge/index.ts#L137-L151

From my understanding, the `balance_transaction` fee should exist for all captured payments. If a payment has been authorised but not captured, we can show the `application_fee_amount`, which should be identical.

I've added a comment with this reasoning in 648f0a1.

Reply:
Thank you for explaining that and the comment is really helpful.